src/main/java/bsh/ConsoleInterface.java

```diff
@@ -43,5 +43,9 @@
     public void println( Object o );
     public void print( Object o );
     public void error( Object o );
+
+    default public void prompt( String prompt ) {
```

Comment:
From the [docs: default methods](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html)
All method declarations in an interface, including default methods, are implicitly public, so you can omit the public modifier.